src/hooks/useArticleHook.js

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import { getSingleArticle } from '../helpers/axiosConfig';
+
+const getArticleById = async (articleId) => {
+  return getSingleArticle(articleId).then((res) => {
+    return res.data;
+  });
+};
+
+export default function useArticleHook(articleId) {
+  return useQuery(['article', articleId], () => getArticleById(articleId));
+}
```

Comment:
There is no need of defining new function in `useQuery`. You can pass an existing function
```suggestion
export default function useArticleHook(articleId) {
  return useQuery(['article', articleId], getArticleById);
}
```
and get `articleId` from the `queryKey` param

Reply:
changed to queryKey